docs/tutorial/part04.rst

```diff
@@ -2,5 +2,437 @@
 How to query the Neo4j database with Neomodel.
 ==============================================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Intro.
+======
+
+I assume that you know how to create a Django project and how to init and setup a Django app inside
+this project. If you don’t have a clue of what I said, I recommend to come back after reading the
+Django intro tutorial and start again from part one of this tutorial:
+https://docs.djangoproject.com/en/1.11/intro/
+
+At this point of the tutorial, you should have already created and setup the paradise_paper_search
+Django project and the fetch_api Django app(:doc:`Part 1 <part01>`). You learned how to integrate
+the Neomodel OGM into the Django project(:doc:`Part 2 <part02>`). Also, at :doc:`Part 3 <part03>`
+you learned the way a Neo4j Graph Database is modeled using Neomodel and ended with a serie of

```

Comment:
I think should be `a series` instead of `a serie`. What do you think? 🤔

Reply:
changed to 'a group'